pixi.toml

```diff
@@ -0,0 +1,77 @@
+[workspace]
+authors = ["Hameer Abbasi <2190658+hameerabbasi@users.noreply.github.com>"]
+channels = ["conda-forge"]
+name = "pytorch-dev"
+platforms = ["linux-64"]
+version = "0.1.0"
+
+[tasks]
+build = "./torch-build.sh"
```

Comment:
`torch-build.sh` will build pytorch core + several domain libraries `torchvision`, `audio` etc. Could we add a task which just does the pytorch core build?

Reply:
Done in https://github.com/Quansight/torch-build/pull/34/commits/564412cfbd3ef768333af158d1a16448603730dc.